app/assets/javascripts/replace_search_text.js

```diff
@@ -0,0 +1,6 @@
+$(document).ready(function() {
+  var width = $(window).width();
+  if(width < 480 ){
+    $(".submit-search-text").text("");
```

Comment:
Ah. Blacklight does this via: https://github.com/projectblacklight/blacklight/blob/v5.13.1/app/assets/stylesheets/blacklight/_header.scss#L31, but the ID is gone now. Pushing up the CSS fix.